coder_sniffer/Drupal/Sniffs/Commenting/InlineVariableCommentSniff.php

```diff
@@ -83,18 +83,27 @@ public function process(File $phpcsFile, $stackPtr)
             if (strpos($tokens[$stackPtr]['content'], '@var') !== false) {
                 $warning = 'Inline @var declarations should use the /** */ delimiters';
 
-                if ($phpcsFile->addFixableWarning($warning, $stackPtr, 'VarInline') === true) {
-                    if (strpos($tokens[$stackPtr]['content'], '#') === 0) {
-                        $varComment = substr_replace(rtrim($tokens[$stackPtr]['content']), '/**', 0, 1)." */\n";
-                    } else if (strpos($tokens[$stackPtr]['content'], '//') === 0) {
-                        $varComment = substr_replace(rtrim($tokens[$stackPtr]['content']), '/**', 0, 2)." */\n";
-                    } else {
-                        $varComment = substr_replace($tokens[$stackPtr]['content'], '/**', 0, 2);
+                if (strpos($tokens[$stackPtr]['content'], '#') === 0 || strpos($tokens[$stackPtr]['content'], '//') === 0) {
+                    if (strpos($tokens[$stackPtr]['content'], '*/') !== false) {
```

Comment:
Did not understand this at first, but we should leave a code comment above like "// If this comment contains '*/' then the developer is mixing inline comment styles. This could be commented out code, so leave this line alone completely."